tests/testutil/publicdomain.go

```diff
@@ -6,6 +6,9 @@ import (
 	"fmt"
 	"strings"
 	"testing"
+	"time"
+
```

Comment:
Nit: Not sure if we have decided how we are grouping imports but I prefer them grouped std lib & other

Reply:
Addressed in person, this is 3 leveled version with ours 3rd, I'll move to 2 level later on.